### DISCUSSION TITLE
[Question] is it possible to have scroll track always hidden?

### DISCUSSION THREAD

**Discussion by xzilja** [idiotWu/smooth-scrollbar#332] [Q&A] [answered]
I wasn't able to find any option for this, closest is `alwaysShowTracks`, but I'm looking for opposite i.e. always hide tracks.
If this is not implemented, would you be open for pr that adds it? If so could you point me to a place in the code where I should add this functionality?

**Comment by dolphin-wood** [comment, marked as answer]
It's a little bit tricky but you can remove the track elements from DOM using:

```js
scrollbar.track.xAxis.element.remove()
scrollbar.track.yAxis.element.remove()
```

or hide it using a simple line of css:

```css
.scrollbar-track { display: none !important }
```

See https://github.com/idiotWu/smooth-scrollbar/issues/206#issuecomment-526831135
